rbd-target-gw.py

```diff
@@ -47,18 +47,18 @@ def ceph_rm_blacklist(blacklisted_ip):
     logger.info("Removing blacklisted entry for this host : "
                 "{}".format(blacklisted_ip))
 
-    result = subprocess.check_output("ceph --conf {cephconf} osd blacklist rm {blacklisted_ip}".
```

Comment:
You should still keep the ```--conf``` while adding ```-n```

Reply:
that's true, I was wrongly assuming that the Ceph conf file would always be `/etc/ceph/ceph.conf`

Reply:
fixed